arlo-client/src/components/Audit/Setup/Participants/_mocks.ts

```diff
@@ -0,0 +1,12 @@
+const jurisdictionFile = new File(
```

Comment:
any way to load that from the `.csv` file that has the same data, so as to minimize duplication?

Reply:
I fixed this in 8a8cecd.